src/set_theory/cardinal/basic.lean

```diff
@@ -968,13 +968,19 @@ theorem lt_aleph_0 {c : cardinal} : c < ℵ₀ ↔ ∃ n : ℕ, c = n :=
   exact ⟨infinite.nat_embedding S⟩
 end, λ ⟨n, e⟩, e.symm ▸ nat_lt_aleph_0 _⟩
 
+theorem lt_aleph_0' {c : cardinal} : c < ℵ₀ ↔ ∃ n : ℕ, ↑n = c :=
```

Comment:
Having symmetric versions of every existential holding an equality doesn't fell like it scales very well. Do we have an established convention here?

Reply:
IMHO, we should either redefine `set.range` and `set.image` to use `∃ ..., y = f x` or use `∃ x, f x = y` everywhere.

Reply:
For now, I'll delete this lemma.